### ISSUE TITLE
Ability to prefix helm oci pushes 

### ISSUE THREAD

**Issue by bryopsida** [eclipse/jkube#2377]
### Component

JKube Kit

### Is your enhancement related to a problem? Please describe

I'd like to be able to push to an oci repo with a prefix in the same way doing the following helm command works.

`helm push *.tgz oci://ghcr.io/<username>/<prefix>`

When running the above command the result is an artifact at `oci://ghcr.io/<username>/<prefix>/<chart-name>`

When I setup my project to do the same with:

```
url = "https://ghcr.io/<username>/<prefix"
```

And run `k8sHelmPush` it results in a not found error during upload.

### Describe the solution you'd like

When defining.

`url = "https://ghcr.io/<username>/<prefix"` as the url it pushes in the same manner `helm push *.tgz oci://ghcr.io/<username>/<prefix>` 

And preserves the original chart name when inspecting with `helm show all oci://ghcr.io/<username>/<prefix>/<chart-name>` in the same way.

**Comment by rohanKanojia** [comment]
@bryopsida : I think you should be able to push helm charts with prefixes now that #2417 is merged.

Could you please test[ `1.15-SNAPSHOT`](https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/jkube/) build to see if your issue is resolved? You can check [how to use jkube snapshots here](https://github.com/eclipse/jkube/blob/master/USING-SNAPSHOT-ARTIFACTS.md#using-snapshots-in-gradle-project).

I tested pushing to GitHub container registry with this configuration (with value of prefix like `username/helm`):

```groovy
kubernetes {
    helm {
        snapshotRepository {
            name = 'GitHub'
            url = "https://ghcr.io/" + System.getenv("HELM_REPO_PATH")
            type = 'OCI'
            username = System.getenv("HELM_REPO_USERNAME")
            password = System.getenv("HELM_REPO_PASSWORD")
        }
    }
}
```

I was able to push helm chart with prefix to this GitHub repository https://github.com/r0haaaan/jkube-helm-chart-upload-ghcr-demo . Could you please check if this is in line with your expectations?

**Comment by bryopsida** [comment]
Sure, I'll try and check it out this weekend.

**Comment by bryopsida** [comment]
@rohanKanojia LGTM, was able to use k8sHelmPush with a prefix and fetch/inspect the latest version with the helm binary.

**Comment by manusa** [comment]
Fixed by #2418
